install/v2plugin/startcontiv.sh

```diff
@@ -1,48 +1,71 @@
 #!/bin/bash
-#Initialize complete contiv container. Start OVS and Net Plugin
 
 set -e 
 
 ### Pre-requisite on the host
-# etcd 
-# sudo modprobe openvswitch
-# sudo mkdir -p /var/contiv
+# run a cluster store like etcd or consul
 
 mkdir -p /var/run/contiv/log
 mkdir -p /var/run/openvswitch
 mkdir -p /etc/openvswitch
 
-echo "V2 Plugin logs" > /var/run/contiv/log/plugin_bootup.log
+BOOTUP_LOGFILE="/var/run/contiv/log/plugin_bootup.log"
 
-echo "Loading OVS" >> /var/run/contiv/log/plugin_bootup.log
-(modprobe openvswitch) || (echo "Load ovs FAILED!!! run 'sudo modprobe openvswitch' from the host before re-enabling the plugin" >> /var/run/contiv/log/plugin_bootup.log && while true; do sleep 1; done)
+echo "V2 Plugin logs" > $BOOTUP_LOGFILE
 
-echo "Starting OVS" >> /var/run/contiv/log/plugin_bootup.log
+if [ $iflist == "" ]; then
+    echo "iflist is empty. Please specify the host interface(s) to use" >> $BOOTUP_LOGFILE
```

Comment:
This should not be mandatory. If only vxlan mode is used, we don't need to specify vlan-if.